provisioning/device/tests/AuthenticationProviderX509Tests.cs

```diff
@@ -1,16 +1,21 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
+﻿using Microsoft.Azure.Devices.Provisioning.Client;
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Move this using statement below

Reply:
just curious, are we actually using Microsoft.Azure.Devices.Provisioning.Client in this test file?

Reply:
No we're not, not sure how that got up there. Removed it